.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
NIT: if this is something I read as a customer, perhaps I'd appreciate a bit friendlier phrasing? 😉

I've asked ChatGPT and it got me something along the lines of
![image](https://github.com/user-attachments/assets/abda6215-cc0b-43f2-98b1-a994b3f24bd5)